samples_project/Assets/Editor/NewAssemblyDef.asmdef

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
If you follow my above suggestion we don't need to add this new assembly definition
> Can this file be put under the samples_project/Assets/SampleViewer/Scripts/ folder?

Reply:
Ah ok, from your comment I see that the Editor folder is special. https://docs.unity3d.com/Manual/SpecialFolders.html

Can you rename this file to `EditorAssemblyDef.asmdef`

Reply:
Will do 👍